src/Optimizely/Entity/Variation.php

```diff
@@ -68,4 +90,18 @@ public function setKey($key)
     {
         $this->_key = $key;
     }
+
+    public function getVariables(){
+        return $this->_variableUsageInstances;
+    }
+
+    public function setVariables($variableUsageInstances){
```

Comment:
Method named like this rather than 'setVariableUsageInstances' because parsed data file has property with tag 'variables' and setters get auto called inside ConfigParser::generateMap